motoman_gp110_support/package.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_gp110_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman GP110 (and variants).</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman GP110 series of manipulators.
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>GP110 - Default</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information 
+      found in the online documentation files for the robot (GP110_2D.zip
+      and GP110_3D.zip from
+      https://www.yaskawa.eu.com/products/robots/handling-mounting/productdetail/product/gp110_704 )
```

Comment:
Is there really no document which provides this information? A datasheet / manual?

If you can, please mention document (file)name and revision, as direct links tend to go stale.

Reply:
`Flyer_Robot_GP110_E_11.2020.pdf` is the closest thing to document name and revision I could find